### PR TITLE
Make submission flash more prominent

### PR DESCRIPTION
**Pull request by louis-vs**
Small CSS adjustment + add a useful & more readable date format.

### CODE REVIEW

File: app/views/spina/conferences/primer_theme/conferences/show.html.haml [hunk 1]
```diff
@@ -5,11 +5,11 @@
         - cache partable_for(:submission_url, parent: @conference) do
           - if @conference.content(:submission_url).present?
             = link_to t(:'.submit_abstract'), @conference.content(:submission_url),
-                      class: %i[btn btn-sm flash-action d-none d-sm-inline-block]
+                      class: %i[btn btn-sm flash-action p-3 p-sm-1]
         - cache partable_for(:submission_date, parent: @conference) do
           %p
```
Comment (jmalcic) [inline app/views/spina/conferences/primer_theme/conferences/show.html.haml:10]: I'm guessing this is to get around the wrapping problem. If the text is `display: inline-block`, then the `flash-action` should float above it more readily, and the button's padding doesn't have to be reduced.
```suggestion
                      class: %i[btn btn-sm flash-action mb-2]
        - cache partable_for(:submission_date, parent: @conference) do
          %p.d-inline-block
```

**Review by jmalcic — APPROVED** [review]
Looks good! I've suggested a small change/alternative for dealing with the wrapping problem

File: app/views/spina/conferences/primer_theme/conferences/show.html.haml (hunk 1 above)
Comment (louis-vs) [reply app/views/spina/conferences/primer_theme/conferences/show.html.haml:10]: Wrapping isn't really a problem though even with small screens, I just wanted to make the button a bit more obvious - I'll remove the padding and switch to `btn-primary` which I think will be fine.

**Comment by jmalcic** [comment]
@louis-vs are you ready for this to be re-reviewed?

**Review by jmalcic — APPROVED** [review]

**Pull request merged**